server/routes/article.routes.js

```diff
@@ -6,5 +6,6 @@ const { articleController } = controllers;
 const router = express.Router();
 
 router.get('/article/:id', articleController.getOneArticle);
+router.delete('/article/:id', articleController.deleteArticle);
```

Comment:
Will there be a future PR to secure these routes?

Reply:
Yes. I would update it while I try to fix this merge conflict.